test/suites/device-mqtt/main_test.go

```diff
@@ -7,13 +7,16 @@ import (
 	"testing"
 )
 
+const snap = "edgex-device-mqtt"
+const snapService = "edgex-device-mqtt.device-mqtt"
```

Comment:
Thanks for the suggestion. I think containing a specific snap name inside these constants will make the test less reuseable. The initial intention for using constants was to accelerate the testing suite development faster for new snaps. Isn't it?

Reply:
It is true that keeping generic names makes it easier to replicate (copy/paste) the tests. But that doesn't make them reusable. Our utility functions are reusable. 

The intention was to make tests easier to write and maintain so that we don't need to write those names many times and possibly make mistakes. 

If we do change to specific names and want to replicate, we can simply rename the variable to match the new target. Most IDEs allow renaming / refactoring a symbol globally by few clicks.

Anyway, I'm fine leaving it as is.

Reply:
I agree with you. Updated!